lib/FreeRTOS/directories.txt

```diff
@@ -0,0 +1,2 @@
+This directory contains Git submodules of the FreeRTOS and "core" libraries built
+by the demo contained in this Git repository.
```

Comment:
We may want to mention that this directory also contains the [mqtt-agent-interface](https://github.com/FreeRTOS/Lab-Project-coreMQTT-Agent/tree/main/lib/FreeRTOS/mqtt-agent-interface) directory, which contains the FreeRTOS implementation of functions in the MQTT agent's interface.

Reply:
Done.